### DISCUSSION TITLE
deleted repositories

### DISCUSSION THREAD

**Discussion by 17N71** [community/community#55882] [New to GitHub]
Hi, I want to know the list of names of all my remote repositories, even the ones that don't show up in settings > remote repositories

Я хочу знать список имен всех моих удаленных репозиториев, даже те которые не отображаются в настройки > удаленные репозитории

**Comment by queenofcorgis** [comment]
In the upper-right corner of any page, click your profile photo, then click Settings.

![image](https://github.com/community/community/assets/92824518/a527bd8d-5a8c-4470-8488-42532c6459f9)

In the "Code planning, and automation" section of the sidebar, click  Repositories.
![image](https://github.com/community/community/assets/92824518/12e6c78c-3c7f-4acf-b906-68ebe37c4c6f)

Under "Repositories", click Deleted repositories.
